pegasus/sites.v3/code.org/homepage.json

```diff
@@ -18,14 +20,50 @@
           "type": "heading",
           "text": "hoc2022_banner_title"
         },
+        {
+          "type": "heading-two",
+          "text": "hoc2022_banner_description"
+        },
+        {
+          "type": "cta_button_solid_yellow",
+          "text": "hoc2022_cta_primary",
+          "url": "/hourofcode2022"
+        },
+        {
+          "type": "cta_button_hollow_white",
+          "text": "hoc2022_cta_secondary",
+          "url": "/hourofcode/overview"
+        }
+      ]
+    },
+    "hoc2022-actual-hoc": {
+      "hoc_modes": ["actual-hoc"],
+      "class": "hoc-banner homepage",
+      "desktopImage": "/images/hoc2022-codeorg-hero-banner-02.png",
+      "mobileImage": "/images/hoc2022-codeorg-hero-banner-mobile-02.png",
+      "leftImage": "/images/hoc2022-codeorg-hero-banner-02-left.svg",
+      "rightImage": "/images/hoc2022-codeorg-hero-banner-02-right.svg",
+      "actions": [
+        {
+          "type": "text",
+          "text": "homepage_hero_hoc_is_here"
```

Comment:
I made a duplicate of the above banner so we can swap out "The Hour of Code is coming" with "The Hour of Code is here" during `actual-hoc`.